inventory.yml

```diff
@@ -0,0 +1,49 @@
+images:
```

Comment:
If this file is being built each time by generate-inventory.sh, should it be added to .gitignore and then removed?